_config.yml

```diff
@@ -678,93 +614,79 @@ t:
-        height: 80
-      -
-        title: "FORCE 06"
-        url: "https://www.departement06.fr/preservation-du-cadre-de-vie/force-06-3080.html"
-        logo: "/img/force06_logo.png"
-        content: "Nuestro socio en el departamento de Alpes Marítimos diversificará las pruebas sobre el terreno"
-        height: 80
-      -
-        title: "SDIS 12"
-        url: "https://www.sdis12.fr/"
-        logo: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSg2bUbkvsTyiTM7uHqbHBY0nxzu7L9AGHEGg&s"
-        content: "Nuestro socio en Aveyron diversificará las pruebas sobre el terreno"
-        height: 125
-      -
-        title: "SDIS 77"
-        url: "https://www.sdis77.fr/"
-        logo: "https://www.infirmiersapeurpompier.com/wp-content/uploads/2020/11/77.jpg"
-        content: "Nuestro socio en Seine&Marne diversificará las pruebas sobre el terreno"
-        height: 75
+        - title: "SDIS 07"
+          url: "https://www.sdis07.fr/"
+          logo: "https://www.sdis07.fr/ardeche-secours-incendie/themes/sapeurs-pompiers-ardeche/medias/img/logo-sdis07-pompiers-ardeche.png"
+          content: "Nuestro primer socio para la recopilación de requisitos y la experimentación de campo"
+          height: 75
+        - title: "Engie Green"
+          url: "https://www.engie-green.fr/"
+          logo: "https://www.engie-green.fr/wp-content/themes/engie-green/assets/img/engie-green-logo.svg"
+          content: "Nuestro primer socio privado, con quien estamos realizando un experimento en Gironde"
+          height: 80
+        - title: "CENIA"
+          url: "https://cenia.cl/"
+          logo: "/img/cenia.jpg"
+          content: "El Centro Nacional de Inteligencia Artificial de Chile con el que estamos trabajando para nuestro primer despliegue en el exterior"
+          height: 60
+        - title: "ADF Subirats"
+          url: "https://subirats.cat/adfsubirats/"
+          logo: "https://i.postimg.cc/NfW4Xyx7/camera-logo.png"
+          content: "Nuestro partner catalán para la adaptación de las cámaras existentes"
+          height: 80
+        - title: "FORCE 06"
+          url: "https://www.departement06.fr/preservation-du-cadre-de-vie/force-06-3080.html"
+          logo: "/img/force06_logo.png"
+          content: "Nuestro socio en el departamento de Alpes Marítimos diversificará las pruebas sobre el terreno"
+          height: 80
+        - title: "SDIS 12"
+          url: "https://www.sdis12.fr/"
+          logo: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSg2bUbkvsTyiTM7uHqbHBY0nxzu7L9AGHEGg&s"
+          content: "Nuestro socio en Aveyron diversificará las pruebas sobre el terreno"
+          height: 125
+        - title: "SDIS 77"
+          url: "https://www.sdis77.fr/"
+          logo: "https://www.infirmiersapeurpompier.com/wp-content/uploads/2020/11/77.jpg"
+          content: "Nuestro socio en Seine&Marne diversificará las pruebas sobre el terreno"
```

Comment:
The new logo is missing in spanish translation, could you point to the new one  as in english ? thks